cmake/externals/get_revisions/CMakeLists.txt

```diff
@@ -0,0 +1,92 @@
+################################################################################
+#
+# medInria
+#
+# Copyright (c) INRIA 2013. All rights reserved.
+# See LICENSE.txt for details.
+# 
+#  This software is distributed WITHOUT ANY WARRANTY; without even
+#  the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#  PURPOSE.
+#
+################################################################################
+
+project(GetRevisions)
+cmake_minimum_required(VERSION 2.8)
+
+## #############################################################################
+## A dummy project configured during an additional step of the medInria project.
+## It retreives the SHA1 and Revision of git and svn based projects,
+## and write the version of all the projects - medinria and its dependeces -
+## into as a macro in medRevisions.h located in the Binary dir of medinria.
+## ############################################################################# 
```

Comment:
Well I have to check. We need on sthg to get the revision for the dev tags in the binary. We used to have an SVN mechanics that is outdated and does not work anymore. It needs to be cleaned up. But in a separate patch please.